src-tauri/src/util.rs

```diff
@@ -143,3 +143,38 @@ pub fn check_northstar_running() -> bool {
         || s.processes_by_name("Titanfall2.exe").next().is_some();
     x
 }
+
+pub fn copy_dir_all(
```

Comment:
Some doc comments (`///`) would be nice explaining what the function does.

It's kinda obvious/guessable based on file name but it's still good practice to have like a one line comment explaining function behaviour to ensure that assumptions are correct ^^